### PR TITLE
Adding clear command and excluding clear, exit from history

### PR DESCRIPTION
**Pull request by jspreddy**
- Added a `clear`, `cls` commands.     #29
- Excluding `clear`, `cls`, `exit` from history.
- Changelog
- Inline help
- Docs

### CODE REVIEW

**Comment by jspreddy** [comment]
Ready for review :)

File: dql/cli.py [hunk 1]
```diff
@@ -759,11 +759,37 @@ def do_EOF(self):  # pylint: disable=C0103
...
+        console.rule()
+        os.system("cls" if os.name in ("nt", "dos") else "clear")
+        self.history_manager.remove_items(n=1)
+
     @repl_command
     def do_exit(self):
         """Exit"""
         self.running = False
         print()
+        self.history_manager.remove_items(n=1)  # remove "exit" from history
```
Comment (stevearc) [inline dql/cli.py:792]: This deletes an item even if I exit with CTRL+D. In the case where I didn't issue any commands, it throws an exception.

File: dql/history.py [hunk 2]
```diff
@@ -55,3 +55,25 @@ def try_to_write_history(self, history_dir: Optional[str] = None) -> None:
...
+    def remove_items(self, n=1):
+        """Remove items from current session's in-memory history."""
+        if n <= 0:
+            return
+
+        try:
+            import readline
+        except ImportError:
+            # Windows doesn't have readline, so gracefully ignore.
+            pass
```
Comment (stevearc) [inline dql/history.py:68]: This should be a `return`, otherwise it will crash on the next line when you use readline

**Review by stevearc — CHANGES_REQUESTED** [review]
Looks good! Just a couple small edge cases to handle.

File: dql/history.py (hunk 2 above)
Comment (jspreddy) [reply dql/history.py:68]: Good catch!

File: dql/cli.py (hunk 1 above)
Comment (jspreddy) [reply dql/cli.py:792]: Will fix.

**Comment by stevearc** [comment]
Thanks!

**Pull request merged**